crates/containerd-shim-wasm/src/sandbox/shim/otel.rs

```diff
@@ -54,12 +56,25 @@ pub struct Config {
 ///
 /// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options
 impl Config {
-    /// Creates a new builder for `OtelConfig`.
+    /// Creates a new builder for `Config`.
     pub fn builder() -> ConfigBuilder {
         ConfigBuilder::default()
     }
 
+    /// Returns `true` if traces are enabled, `false` otherwise.
+    ///
+    /// Traces are enabled if either `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` or `OTEL_EXPORTER_OTLP_ENDPOINT` is set and not empty.
+    pub fn traces_enabled() -> bool {
```

Comment:
**suggestion:** make this a member method instead of a static method, then you can use `self.traces_endpoint`
```suggestion
    pub fn traces_enabled(&self) -> bool {
```

Reply:
I actually made `traces_enabled` a function instead of a method, since the complexity of it increases after I added `OTEL_SDK_DISABLED`